src/main/java/com/spotify/futures/CompletableFutures.java

```diff
@@ -594,6 +632,48 @@ public static void checkCompleted(CompletionStage<?> stage) {
                                                ff.join()));
   }
 
+  /**
+   * Composes multiple stages into another stage using a function.
+   *
+   * @param function the combining function.
+   * @param stages   the stages to combine
+   * @param <T>      the type of the combining function's return value.
+   * @return a stage that is composed from the input stages using the function.
+   * @since 0.4.0
+   */
+  public static <T> CompletionStage<T> combineFutures(
+          Function<Combined, CompletionStage<T>> function, CompletionStage<?>... stages) {
```

Comment:
I'm confused by the suffix -`Futures` here, what does it mean?

In the `CompletionStage` class, this is typically called a `compose` operation, so maybe `combineCompose`?

In other cases, this is typically called a `flatMap` operation, so maybe `combineFlatMap`? (I don't personally like that one)

Reply:
Ah, I see that we already have methods called `combineFutures` that were added without me noticing :) So I guess there is some precedent

Reply:
Yes, perhaps those other ones were poorly named :(